test/utility/test_util.i

```diff
@@ -108,8 +111,11 @@ testutil_clean_work_dir(char *dir)
 		testutil_die(ENOMEM, "Failed to allocate memory");
 
 	snprintf(buffer, inputSize, "%s%s", RM_COMMAND, dir);
-	if ((ret = system(buffer)) != 0)
-		testutil_die(ret, "System call to remove directory failed");
+
+	if ( stat( dir, &info ) == 0 )
```

Comment:
The spacing still isn't right here. There should not be a space before ```stat``` or after the ```0```.